webapp/api/marketo.py

```diff
@@ -84,7 +84,7 @@ def get_user(self, email):
         response = self.request(
             method="GET", url=LEAD_BY_EMAIL, url_args=dict(email=email)
         )
-        return response["result"][0]
+        return response["result"][0] if len(response["result"]) > 0 else {}
```

Comment:
While we're here we should probably check response is there too:
`return response["result"][0] if "result" in response and len(response["result"]) > 0 else {}`?

Reply:
@solazio why an empty dict instead of `None`?

Reply:
@Lukewh yes, it makes sense to check if it exists as well. thanks